modules/ROOT/pages/server-compatibility-eventing.adoc

```diff
@@ -53,12 +53,8 @@ You can use Eventing to handle data changes that happen when applications intera
 
 You can also use Eventing to generate vector embeddings from document fields, see xref:couchbase-lite:ROOT/cbl-whatsnew.adoc#vector-search[Vector Search] for more details about Vector Search with Couchbase Lite.
```

Comment:
@simon-dew the commented section at Line 51. Is that functionality now available in 7.6.4 or has it been moved to 7.6.5?

Reply:
No I think that functionality is available in 7.6.4 — you can uncomment those lines

Reply:
Done!